src/server/connection.rs

```diff
@@ -384,6 +386,7 @@ impl Connection {
             file_remove_log_control: FileRemoveLogControl::new(id),
             #[cfg(feature = "gpucodec")]
             supported_encoding_flag: (false, None),
+            user_session_id: None,
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/devnotes/getting-the-session-id-of-the-current-process
You can try this if you didn't, and confirm if the user still needs to reconnect if they select the same session ID.

Reply:
Got it, Thanks.